import "light_status.proto";

option java_package = "com.nextev.cvs_proto.protobuf.events";
option java_outer_classname = "LightChangeMessage";

message LightChangeEvent {

    optional string id = 1; // unique id, VIN for vehicle data
    optional int32 version = 2; // message protocol version
    optional uint64 sample_ts = 3; // light status change time
    optional LightStatus light_status = 4;
}